source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md

```diff
@@ -9,9 +9,9 @@ ______________________________________________________________________
 
 Presently, seeding a driver with an initial list of ReplicaSet or MongoS addresses is somewhat cumbersome, requiring a
 comma-delimited list of host names to attempt connections to. A standardized answer to this problem exists in the form
-of SRV records, which allow administrators to configure a single domain to return a list of host names. Supporting this
-feature would assist our users by decreasing maintenance load, primarily by removing the need to maintain seed lists at
-an application level.
+of SRV records, which allow administrators to configure a single hostname to return a list of host names. Supporting
```

Comment:
It's not really a hostname either... Uhhh "... a single record.." or "single resource record"?

Reply:
I changed it to 'single SRV record', I think thats what's supposed to be conveyed.